### ISSUE TITLE
Historical player positions

### ISSUE THREAD

**Issue by flipperbw** [BurntSushi/nfldb#240]
Would it be possible to add this to the database? It's tough querying for older or changed players when they move positions, or get listed as UNK after playing years at a certain position. It would also help for when a WR throws a pass during a game (if you only want true QB stats).

Thanks.

**Comment by BurntSushi** [comment]
Basically, my answer is the same as it is for nflgame. It is possible, but difficult. The reasons why it is difficult are:

1. The player database corresponds to a snapshot in time. Adding historical data means converting the database to time series data, which is a *huge* change.
2. Actually getting historical player positions would require either finding a different data source for it, or building the history ourselves. The latter would be difficult and would take years for it to become meaningful.

In short, I don't think this is a reasonable request to make. If you want historical data, you need to go out and build it yourself. Either that, or figure out a different way to solve whatever problem you're having. For example, you could heuristically guess the position of a player based on their statistical events.

**Comment by flipperbw** [comment]
Posted here accidentally before posting in nflgame. Moving the convo there.
